src/ScottPlot/Plottable/RadialGaugePlot.cs

```diff
@@ -0,0 +1,709 @@
+                AngleSwept = (_GaugeDirection == RadialGaugeDirection.AntiClockwise ? -1 : 1) * (float)(_AngleRange * DataRaw[i] / (_MaxScale - _MinScale));
+
+
+                DataAngular[i, 0] = (_GaugeMode == RadialGaugeMode.Stacked ? _StartingAngleGauges : AngleInit);
+                DataAngular[i, 1] = AngleSwept;
+
+                if (DataRaw[i] >= 0)
+                    AngleSumPos += AngleSwept;
+                else
+                    AngleSumNeg += AngleSwept;
+            }
+
+            // Compute the initial angle for the background gauges
+            _StartingAngleBackGauges = _StartingAngleGauges;
+            _StartingAngleBackGauges += (_GaugeDirection == RadialGaugeDirection.AntiClockwise ? -1 : 1) * (float)(_AngleRange * _MinScale / (_MaxScale - _MinScale));
+        }
+
+        /// <summary>
+        /// Sets the value of <see cref="MaxScale"/> and <see cref="MinScale"/> properties
+        /// </summary>
+        private void ComputeMaxMin()
+        {
+            if (GaugeMode == RadialGaugeMode.Sequential || GaugeMode == RadialGaugeMode.SingleGauge)
+            {
+                _MaxScale = DataRaw.Sum(x => Math.Abs(x));
+                _MinScale = 0;
+            }
+            else
+            {
+                _MaxScale = DataRaw.Max(x => Math.Abs(x));
+                var min = DataRaw.Min();
+                _MinScale = min < 0 ? min : 0;
+            }
+        }
+
+        /// <summary>
+        /// Needed as part of IPlottable in ScottPlot.ScottForm
+        /// </summary>
+        /// <param name="deep"></param>
+        public void ValidateData(bool deep = false)
+        {
+            if (GaugeLabels != null && GaugeLabels.Length != DataRaw.Length)
+                throw new InvalidOperationException("Gauge labels must match size of data values");
+        }
+
+        /// <summary>
+        /// Needed as part of IPlottable in ScottPlot.ScottForm
+        /// </summary>
+        /// <returns></returns>
+        public LegendItem[] GetLegendItems()
+        {
+            if (GaugeLabels is null)
+                return null;
+
+            List<LegendItem> legendItems = new List<LegendItem>();
+            for (int i = 0; i < GaugeLabels.Length; i++)
+            {
+                var item = new LegendItem()
+                {
+                    label = GaugeLabels[i],
+                    color = GaugeColors[i],
+                    lineWidth = 10,
+                    markerShape = MarkerShape.none
+                };
+                legendItems.Add(item);
+            }
+
+            return legendItems.ToArray();
+        }
+
+        /// <summary>
+        /// Needed as part of IPlottable in ScottPlot.ScottForm
+        /// </summary>
+        /// <returns></returns>
+        public AxisLimits GetAxisLimits() =>
+            (GaugeLabels != null) ? new AxisLimits(-3.5, 3.5, -3.5, 3.5) : new AxisLimits(-2.5, 2.5, -2.5, 2.5);
+
```

Comment:
Why does this depend on whether labels are present or not?

Reply:
This was copy-pasted from `RadarPlot`. This is needed as part of the IPlottable interface, and I don't know how to handle it here.

Reply:
This method describes how large the plottable is on the coordinate system so when the user auto-scales (by calling `AxisAuto()` or middle-clicking a user control the axis limits are adjusted to include these dimensions.

I'll fix this up shortly 👍 I think the axis limits for this plot type depend on the number of groups and the line widths for each group